src/pages/about-us/index.jsx

```diff
@@ -1,13 +1,10 @@
 import React from 'react';
-//import styles 
-
-export default function About()
-{
-    return(
+import flippingCard from '@/components/about-us/flippingCard';
+export default function About() {
+    return (
         <>
-        about yay
+            <flippingCard>Card1,Card2,Card3,Card4,Card5</flippingCard>
```

Comment:
Make sure to edit the name of the import to `FlippingCard` as well as the component. Also, you can start to send in your props here. Since each card is exactly the same, I would recommend using the `map` function in JS to create all flipping cards at once. Something like this:
```
/* Define an array of cards: 1, 2, ... 5 up top */

array.map((elem) => {
  return <FlippingCard content={t('Card' + index + '.content')} ...> </FlippingCard>
})
```
There might be a more efficient way of mapping the cards so you are free to experiment if you wish. Using map will reduce the workload by ALOT